### PR TITLE
style: Home-page, UI only

### PR DESCRIPTION
**Pull request by tushartiwari7**
# Built Landing Page 
Issue #2 
Some files were reviewed in a previous [PR](https://github.com/tushartiwari7/furncastle/pull/1) of another Repo.

### Categories
  - [Categories.jsx](https://github.com/tushartiwari7/furn-house/compare/landing-page?expand=1#diff-799e2840109083d50d993eb8c6c07fed74fbcdf6ab1c1c641a5e5875ce9677a9)
  - [Categories.css](https://github.com/tushartiwari7/furn-house/compare/landing-page?expand=1#diff-a29f4b689048b1cc0e4a6e867efd27e330dfe7207fa7eee6a789a082e081d232)
  - [db/categories.js](https://github.com/tushartiwari7/furn-house/compare/landing-page?expand=1#diff-2db1cd5d79f30b1a054d1e09e7ece5f06e14ed90681a676b54c1eaabf726c638)
### Banner
- [Banner.css](https://github.com/tushartiwari7/furn-house/compare/landing-page?expand=1#diff-006ccdb610ee301c2c880dea07856d0763468cc16387211191719c665c66c78e)
- [Banner.jsx](https://github.com/tushartiwari7/furn-house/compare/landing-page?expand=1#diff-b5820984c1f147e5f2c7a35b5603aea615b52ba60a8bf90234bf60d0ef978772)

localhost_3000_ (1)

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import './Header.css';
+
+export const Header = () => {
+  return (
+    <header className="full-width p-sm flex header fs-l">
+      <div className="flex flex-center">
+        <button id="hamburger" className="btn btn-float btn-outline-primary hamburger mx-xs rounded-circle flex flex-center">
+          <i className="mx-xs bi bi-list"></i>
+        </button>
+        <a className="list white flex flex-center" href="/">
```
Comment (Priya-Patel31) [inline src/components/Header/Header.jsx:11]: fix : change to link tag

File: src/components/Banner/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const Banner = () => {
+  return (
+    <Link to='/products'>
+      <img src="https://godrejinterio.com/imagestore/B2C/EspotImages/Images/Banners/Q4_HomePage-01.webp" alt="Sofa_Sets"
+      width={'100%'}
+      className="banner-img"
+      ></img>
```
Comment (Priya-Patel31) [inline src/components/Banner/Banner.jsx:10]: fix : img is self closing tag

**Review by Priya-Patel31 — APPROVED** [review]
Loved the way you have broken your code into small components . Just have few minor changes suggested to you.

**Review by Raghav888 — COMMENTED** [review]
Approved
Good Naming convention

**Review by Raghav888 — APPROVED** [review]

**Pull request merged**